src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -110,6 +110,32 @@
   col-sm input-container
 {%- endblock form_group_class %}
 
-{% block form_row_class -%}
-  form-group row{% if row_attr.class is defined %} {{ row_attr.class }}{% endif %}
-{%- endblock form_row_class %}
+{% block form_row_attr -%}
```

Comment:
This is a good idea, however, you should keep the `form_row_class` block and include it because some modules extend it It's an easy way to extend our UI kit and change all the rows' classes at once